### ISSUE TITLE
Could you please disclose the specific case divisions you used in the dataset, training set, validation set and test set

### ISSUE THREAD

**Issue by hhjjaaa** [MQinghe/MiDSS#11]
Could you please disclose the specific case divisions you used in the dataset, training set, validation set and test set
